### ISSUE TITLE
Generating UIImage extension with cases corresponding to images that are no longer in my Images.xcassets folder

### ISSUE THREAD

**Issue by muuhoffman** [AliSoftware/SwiftGen#107]
Where is this generating the cases from for the UIImage extension.  The tool is generating cases that correspond to images that I have removed from Images.xcassets.  My current images show up in alphabetical order, but the removed images show up after those images in alphabetical order.

i.e.
    // current images
    case More = "more"
    case No = "no"
    case Refresh = "refresh"
    case Unchecked = "unchecked"
    case User = "user"
    case Users = "users"
    case X = "x"
    // deleted images
    case Back = "back"
    case Envelope = "envelope"
    case Forward = "forward"
    case Settings = "settings"

**Comment by AliSoftware** [comment]
Hi

If you browse to your Assets Catalog in the Finder, do the '.imageset` folders for the deleted images still exists (probably empty)?

If so, does deleting those empty folders solves the issue?

**Comment by muuhoffman** [comment]
I figured it out.  I had two Images.xcassets folders.  Since I'm using Cocoapods, I'm in a workspace.  In Xcode, I only see one Images.xcassets folder with my desired images.  However, my project (.xcodeproj) file/folder had an Images.xcassets folder with the images I had deleted at some point in the past in it.  Removing this old Images.xcassets folder fixed my issue.

To find these pesky images I used this command
`find . -name "*.png" | grep <name of old image>`
